app/models/concerns/user_search.rb

```diff
@@ -18,10 +18,10 @@ module UserSearch
     }
 
     def self.search_user(role_name, search_query, search_text)
-      joins(:roles)
-      .where('roles.name = ?', role_name)
-      .where(search_query, search_text: "%#{search_text}%")
-      .distinct
+      res = joins(:roles)
```

Comment:
@steveyken @abhinav098 I've added this change to this PR.

Since donors don't have a specific roles, we weren't able to find them through the search endpoint. So I made `role_name` optional.  Do you foresee any unexpected impact this could have ?